src/main/java/dk/cngroup/lentils/service/CypherGameInfoService.java

```diff
@@ -21,7 +23,10 @@ public CypherGameInfoService(final CypherGameInfoRepository cypherGameInfoReposi
         return cypherGameInfoRepository.findAll();
     }
 
-    public List<CypherGameInfo> getAllByTeamId(final Long teamId) {
-        return cypherGameInfoRepository.findAllByTeamId(teamId);
+    public List<CypherGameInfo> getAllByTeamIdAndStatusIsNotLocked(final Long teamId) {
+        int statusLocked = CypherStatus.valueOf("LOCKED").ordinal();
```

Comment:
Slo by to jeste napsat takhle?
CypherStatus.LOCKED.ordinal()

A zbavit se zase toho stringu?